next-app/src/common/common-types.ts

```diff
@@ -4,4 +4,6 @@ export type Maybe<T> = T | Nil;
 
 export type Id = number;
 
+export type Id2 = number;
```

Comment:
why not just use Id ?

Reply:
When I tried to fix the routing error, I have changed Id > Id2, I can re change it to Id

Reply:
> When I tried to fix the routing error, I have changed Id > Id2, I can re change it to Id

No need, understood, I will get it during my weekly file clean up.